Pinta.Tools/Editable/EditEngines/ArrowedEditEngine.cs

```diff
@@ -27,33 +27,32 @@
 using System;
 using System.Collections.Generic;
 using Cairo;
-using Gtk;
 using Pinta.Core;
 
 namespace Pinta.Tools;
 
 public abstract class ArrowedEditEngine : BaseEditEngine
 {
-	private Separator? arrow_sep;
-	private Label? arrow_label;
-	private CheckButton? show_arrow_one_box, show_arrow_two_box;
+	private Gtk.Separator? arrow_sep;
```

Comment:
For future PRs, please avoid making extra changes like this which make it harder to review the real changes